apps/dolly-frontend/src/main/js/src/components/fagsystem/arena/form/partials/MedServicebehov.js

```diff
@@ -1,14 +1,57 @@
-import React from 'react'
+import React, { useContext } from 'react'
 import { Kategori } from '~/components/ui/form/kategori/Kategori'
 import { SelectOptionsManager as Options } from '~/service/SelectOptions'
 import { FormikSelect } from '~/components/ui/form/inputs/select/Select'
 import { FormikDatepicker } from '~/components/ui/form/inputs/datepicker/Datepicker'
+import _get from 'lodash/get'
+import { Alert } from '@navikt/ds-react'
+import { BestillingsveilederContext } from '~/components/bestillingsveileder/Bestillingsveileder'
+
+const errorPaths = [
+	`arenaforvalter.aap115[0].fraDato`,
+	`arenaforvalter.aap[0].tilDato`,
+	`arenaforvalter.dagpenger[0].tilDato`,
+]
+
+const feilmelding25 = 'Vedtak kan ikke overlappe med 25-årsdagen til person. '
+const feilmelding67 = 'Person kan ikke ha vedtak etter fylte 67 år. '
+const feilmeldingFiks =
+	'Vennligst endre vedtaksperioden eller sett en spesifikk alder/fødsel på person for å unngå denne feilen.'
+
+const getFeilmelding = (formikBag) => {
+	let har25Feil = false
+	let har67Feil = false
+	for (let path of errorPaths) {
+		const feil = _get(formikBag.errors, path)
+		if (feil && feil.includes('25')) {
+			har25Feil = true
```

Comment:
Her og i `else if` under kan vel `return melding += feilmelding25` brukes direkte for å unngå let over og kodebiten under kan vel da byttes med bare `return null` 😄

Reply:
Nei, fordi flere paths kan ha 25 års feilen. Så da blir det duplikat tekst. Men kan ta en titt på å forenkle det.

Reply:
Fikk forenklet det litt nå :)

Reply:
Ser supert ut! 😄